package.json

```diff
@@ -13,7 +13,7 @@
     "changelog": "npm run changelog:generate && npm run changelog:add",
     "changelog:add": "git add CHANGELOG.md",
     "changelog:generate": "github_changelog_generator --future-release $npm_package_version",
-    "coverage": "nyc --no-clean mocha",
+    "coverage": "nyc --no-clean jest --coverage",
```

Comment:
We don't need nyc here, we don't have extra process (i.e. `child_process`/`execa`/etc), so let's remove it